Libs/MRML/Logic/vtkMRMLApplicationLogic.cxx

```diff
@@ -312,6 +312,93 @@ void vtkMRMLApplicationLogic::PropagateVolumeSelection(int fit)
     }
 }
 
+//----------------------------------------------------------------------------
+void vtkMRMLApplicationLogic::PropagateBackgroundVolumeSelection(int fit)
```

Comment:
I want the separate methods in the API, but I want to avoid the duplication in the code.  So the only confusion is I didn't explain that well : )